### ISSUE TITLE
show RL reward on plot graph

### ISSUE THREAD

**Issue by CryptoRahino** [freqtrade/freqtrade#8689]
## Describe your environment
(if applicable)

  * Operating system: ubuntu 22
  * Python Version: 3.10.0 (`python -V`)
  * CCXT version: ccxt==3.0.97 (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2023.5.dev-5cab19ed4 (`freqtrade -V` or `docker compose run --rm freqtrade -V` for Freqtrade running in docker)

## Describe the enhancement
Hey,
currently the RL reward can be seen only in tensor board, and the trades on plot.
when researching the output of backtests it is useful to see the correlation between the reward and the trade on the same graph.
please let me know if further information is required

**Comment by xmatthias** [comment]
Plotting (plot-dataframe, freqUI) can plot whatever the strategy puts into the dataframe as part of the strategy (obviously, correctly aligned to the candles).

The reward will not be part of that - and i also see no straightforward way to do this - especially not considering re-trainings throughout backtesting.
Tensorflow is made for especially this - and we don't strive to re-implement that functionality - so i currently don't see this as something builtin (unless @robcaulk  tells me it's straightforward to add this information as new column to the dataframe automatically).

**Comment by robcaulk** [comment]
The question really is "which reward?"

Reward is computed in the training environment and is used as part of the loss function for updating weights to train your neural network. When you inference your trained network to get an action for an actual trade (which is what FreqUI is showing you), there is no reward computed. Therefore, I dont know which reward you are expecting to plot there.

**Comment by xmatthias** [comment]
Going to close this based on the above answers (and the subsequent inactivity).
